### ISSUE TITLE
./local/test.sh: line 11: 11996 Aborted                 (core dumped) python3 ${BIN_DIR}/test.py --config=${config_path} --checkpoint=${train_output_path}/checkpoints/${ckpt_name}

### ISSUE THREAD

**Issue by dwaylin** [PaddlePaddle/Paddle#51672]
### 请提出你的问题 Please ask your question

测试：
./run.sh --stage 2 --stop-stage 2
提示：
`(base) root@1:/home/a/ldw/PaddleSpeech/examples/iwslt2012/punc0# ./run.sh --stage 2 --stop-stage 2
/root/miniconda3/lib/python3.10/site-packages/_distutils_hack/__init__.py:33: UserWarning: Setuptools is replacing distutils.
  warnings.warn("Setuptools is replacing distutils.")
========Args========
checkpoint: exp/default/checkpoints/snapshot_iter_12840.pdz
config: conf/default.yaml
ngpu: 1
print_eval: true

========Config========
batch_size: 64
data_params:
  pretrained_token: ernie-1.0
  punc_path: data/iwslt2012_zh/punc_vocab
  seq_len: 100
dataset_type: Ernie
dev_path: data/iwslt2012_zh/dev.txt
max_epoch: 20
model:
  num_classes: 4
  pretrained_token: ernie-1.0
model_type: ErnieLinear
num_snapshots: 10
num_workers: 2
optimizer_params:
  weight_decay: 1e-06
scheduler_params:
  gamma: 0.9999
  learning_rate: 1e-05
seed: 42
test_path: data/iwslt2012_zh/test.txt
train_path: data/iwslt2012_zh/train.txt
[2023-03-15 08:30:30,207] [    INFO] - Already cached /root/.paddlenlp/models/ernie-1.0/vocab.txt
[2023-03-15 08:30:30,214] [    INFO] - tokenizer config file saved in /root/.paddlenlp/models/ernie-1.0/tokenizer_config.json
[2023-03-15 08:30:30,214] [    INFO] - Special tokens file saved in /root/.paddlenlp/models/ernie-1.0/special_tokens_map.json
Preprocessing in PuncDatasetFromErnieTokenizer...
[2023-03-15 08:30:32,459] [    INFO] - Model config ErnieConfig {
  "attention_probs_dropout_prob": 0.1,
  "enable_recompute": false,
  "fuse": false,
  "hidden_act": "relu",
  "hidden_dropout_prob": 0.1,
  "hidden_size": 768,
  "id2label": {
    "0": "LABEL_0",
    "1": "LABEL_1",
    "2": "LABEL_2",
    "3": "LABEL_3"
  },
  "initializer_range": 0.02,
  "intermediate_size": 3072,
  "label2id": {
    "LABEL_0": 0,
    "LABEL_1": 1,
    "LABEL_2": 2,
    "LABEL_3": 3
  },
  "layer_norm_eps": 1e-12,
  "max_position_embeddings": 513,
  "model_type": "ernie",
  "num_attention_heads": 12,
  "num_hidden_layers": 12,
  "pad_token_id": 0,
  "paddlenlp_version": null,
  "pool_act": "tanh",
  "task_id": 0,
  "task_type_vocab_size": 3,
  "type_vocab_size": 2,
  "use_task_id": true,
  "vocab_size": 18000
}

W0315 08:30:33.456214 11996 gpu_resources.cc:61] Please NOTE: device: 0, GPU Compute Capability: 7.5, Driver API Version: 11.2, Runtime API Version: 11.2
W0315 08:30:33.457975 11996 gpu_resources.cc:91] device: 0, cuDNN Version: 8.1.
[2023-03-15 08:30:33,961] [ WARNING] - Some weights of the model checkpoint at ernie-1.0 were not used when initializing ErnieForTokenClassification: ['cls.predictions.layer_norm.bias', 'cls.predictions.transform.weight', 'cls.predictions.transform.bias', 'cls.predictions.decoder_bias', 'cls.predictions.layer_norm.weight']
- This IS expected if you are initializing ErnieForTokenClassification from the checkpoint of a model trained on another task or with another architecture (e.g. initializing a BertForSequenceClassification model from a BertForPreTraining model).
- This IS NOT expected if you are initializing ErnieForTokenClassification from the checkpoint of a model that you expect to be exactly identical (initializing a BertForSequenceClassification model from a BertForSequenceClassification model).
[2023-03-15 08:30:33,962] [ WARNING] - Some weights of ErnieForTokenClassification were not initialized from the model checkpoint at ernie-1.0 and are newly initialized: ['ernie.embeddings.task_type_embeddings.weight', 'classifier.weight', 'classifier.bias']
You should probably TRAIN this model on a down-stream task to be able to use it for predictions and inference.
W0315 08:30:35.013073 11996 gpu_resources.cc:217] WARNING: device: . The installed Paddle is compiled with CUDNN 8.2, but CUDNN version in your machine is 8.1, which may cause serious incompatible bug. Please recompile or reinstall Paddle with compatible CUDNN version.
./local/test.sh: line 11: 11996 Aborted                 (core dumped) python3 ${BIN_DIR}/test.py --config=${config_path} --checkpoint=${train_output_path}/checkpoints/${ckpt_name}
`

知道这是什么原因，怎么解决吗？

**Comment by zmxdream** [comment]
你好，可以到PaddleSpeech下提issue, https://github.com/PaddlePaddle/PaddleSpeech
